docs/workflows/syntax/functions.mdx

```diff
@@ -119,6 +119,28 @@ steps:
 
 ```
 
+### `render_without_execution`
+
+**Description:** Renders the string without execution of keep instructions inside this string.
+**Example:**
+
+```yaml
+consts:
+  yaml: |
+    keep.is_business_hours(2024-03-25T14:00:00Z)
```

Comment:
its not valid yaml

Reply:
<img width="810" alt="Screenshot 2025-01-15 at 14 46 52" src="https://github.com/user-attachments/assets/961a7eae-99cf-4849-a0d6-cda0a40a0978" />